examples/passport-login/data/db.json

```diff
@@ -1,15 +1,22 @@
 {
   "ids": {
-    "User": 6,
-    "UserIdentity": 104566694532673,
-    "UserCredentials": 3
+    "User": 9,
+    "UserIdentity": 104566694532676,
+    "UserCredentials": 4
```

Comment:
Is it possible for the this file to zero entries in User, UserIdentity, and UserCredentials, and have the "ids" have entries with value 1 or something. Right now the "ids".UserCredentials is 4, but looking in UserCredentials, the userId is 6. So it looks weird.

Reply:
@emonddr I have enabled clearing test data after every test run